README.md

```diff
@@ -64,326 +67,20 @@ yarn add @xmtp/react-native-sdk
 
 ### Configure for iOS
 
-In the `ios` directory, update your `Podfile` file as follows:
+1. In the `ios` directory, update your `Podfile` file to set this value: `platform :ios, '16.0'`. This is required by XMTP.
```

Comment:
Is the flow in this section correct? Just making sure I structured it correctly. TY!